.github/WORKFLOW_ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,8 @@
+---
+title: 'Workflow "{{ env.WORKFLOW_NAME }}" failed (run_id: {{ env.RUN_ID }})'
+---
+
+The workflow "**{{ env.WORKFLOW_NAME }}**" failed. See logs:
+https://github.com/{{ env.REPO_SLUG }}/actions/runs/{{ env.RUN_ID }}
+
+// cc: {{ env.MENTION }}
```

Comment:
Should we assign the issue instead/as well as mention?

Reply:
And add a reminder to close the issue when resolved.

Reply:
> Should we assign the issue instead/as well as mention?

We can't assign to teams, only to individuals... and doing so would mean a lot more code to extract individuals and do some sort of round-robin assignment, etc. Unless you have a concrete idea on how to decide this, I think we can punt for now and decide later if it's worth the effort?